Gems/Atom/Feature/Common/Assets/ShaderLib/Atom/Features/BlendUtility.azsli

```diff
@@ -45,10 +45,13 @@ float3 ApplyTextureBlend(float3 color, float3 blendColor, float factor, TextureB
     {
         // [GFX TODO][ATOM-3237] See if we can avoid these extra color space conversions...
         // To get blend behavior that mimics photoshop, convert to sRGB
-        float3 blendColorSRGB = TransformColor(blendColor, ColorSpaceId::ACEScg, ColorSpaceId::SRGB);
-        float3 colorSRGB = TransformColor(color, ColorSpaceId::ACEScg, ColorSpaceId::SRGB);
+        //Acessg -> srgb
+        real3 blendColorSRGB = AcesCg_To_LinearSrgb(blendColor);
+        blendColorSRGB = LinearSrgb_To_Srgb(blendColorSRGB);
```

Comment:
For blending, don't we want to stay in linear SRGB?

Reply:
I didnt want to change the existing behavior in this case. This could be a separate change in future.